Automattic/jetpack
# Instant Search: Remove redundant links in results.

Fixes #25698.

#### Changes proposed in this Pull Request:


Specifically in the expanded results. The links are still visible and work as expected but they are no longer part of the tab loop or visible to screen readers (the contexts in which they are redundant).

Adding `tabindex="-1"` removes the elements from the tab loop while adding `aria-hidden="true"` hides the elements from screen readers. Both are needed to satisfy a11y standards.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit a site with a Jetpack Search plan.
2. Open the Overlay by appending `?s=&category=uncategorized` to your site address.
3. Confirm the title, path, and image are all clickable in the results.
4. Tab into the search results with the keyboard.
5. Confirm that only the title is accessible via keyboard navigation.

#### For screen readers:

1. Open the Accessibility panel in Firefox.
2. Select the "Check for issues: Keyboard" option.
3. Confirm their are no issues flagged in the search results.

**Note:** To enable full keyboard navigation in Firefox, turn on this checkbox:

![FullKeyboardNavSetting](https://user-images.githubusercontent.com/40267301/185374691-f1d13cc5-7902-4b15-b42b-842cfaffc52d.png)
